Automattic/newspack-blocks
# feat(donate): support empty value for "other" tier

### All Submissions:






### Changes proposed in this Pull Request:

1200550061930446-as-1205933703541044

Deprecates the customization of the default "Other" value for the Donate block when using the tiered layout. The value should now always start empty so the donor can decide their donation amount with less bias.

While empty, the "Donate now" button should be greyed out and disabled.

#### Editor

| Before | After |
| --- | --- |
| <img width="1162" alt="image" src="https://github.com/Automattic/newspack-blocks/assets/820752/af87398e-bfa6-454e-8109-35ac6907ec7e"> | <img width="1165" alt="image" src="https://github.com/Automattic/newspack-blocks/assets/820752/36d7885c-f5d6-4c19-9d34-18bb9ebc504b"> |

#### Block render

| Empty | Filled |
| --- | --- |
| <img width="719" alt="image" src="https://github.com/Automattic/newspack-blocks/assets/820752/ce0c5b8b-583a-44d3-a8fc-96562a903524"> | <img width="717" alt="image" src="https://github.com/Automattic/newspack-blocks/assets/820752/fe5c41b5-791a-4e98-a2d2-5afff56ee1e6"> |

### How to test the changes in this Pull Request:

1. Before switching to this branch, make sure you have a tiered Donate block configured
2. Check out this branch, visit the page containing the block, switch to the "Other" option on every frequency, and confirm it renders empty with the button greyed out, as in the image above
3. Enter a value and confirm you are able to donate the desired amount
4. Edit the page and confirm the block editor no longer includes an input to edit the "Other" tier

### Other information:
